resources/scripts/components/server/Console.tsx

```diff
@@ -81,6 +81,17 @@ export default () => {
             // @see https://github.com/xtermjs/xterm.js/issues/2265
             // @see https://github.com/xtermjs/xterm.js/issues/2230
             TerminalFit.fit(terminal);
+
+            // Add support for copying terminal text.
+            terminal.attachCustomKeyEventHandler((e: KeyboardEvent) => {
+                // Ctrl + C
+                if (e.ctrlKey && (e.key === 'c')) {
+                    document.execCommand('copy');
+                    return false;
```

Comment:
isn't just the `return false` sufficient?

Reply:
From what I understand, the `return false` prevents xterm.js from processing the event, while `return true` allows all additional events to be handled.  So while the extra return might not be necessary because xterm.js does not do anything observable with `Ctrl+C`, it should probably be left there.

https://xtermjs.org/docs/api/terminal/classes/terminal/#attachcustomkeyeventhandler

Reply:
I'd assume if you prevent xterm from processing it by returning false, it shouldn't consume the event. It therefore propagates until the browser itself consumes it and copies the text. Just listening for keyboard events doesn't disable all of them, does it?